vignettes/quick-get.qmd

```diff
@@ -0,0 +1,252 @@
+---
+title: "Quicky get daily data"
+vignette: >
+  %\VignetteIndexEntry{Quicky get daily data}
+  %\VignetteEngine{quarto::html}
+  %\VignetteEncoding{UTF-8}
+bibliography: references.bib
+number-sections: true
+format:
+  html:
+    toc: true
+    toc-depth: 2
+    code-overflow: wrap
+execute: 
+  eval: false
+---
+
+# Introduction {#intro}
+
+This vignette demonstrates how to get minimal daily aggregated data on the number of trips between municipalities using the `spod_quick_get_od()` function. With this function, you only get total trips for a single day, and no additional variables that are available in the full [v2 (2022 onwards) data set](v2-2022-onwards-mitma-data-codebook.html). The advantage of this function is that it is much faster than downloading the full data from source CSV files using `spod_get()`, as each CSV file for a single day is about 200 MB in size.
```

Comment:
And uses less memory. But that's kind of self-explanatory. Great intro.

Reply:
@Robinlovelace Actually, it may not be obvious to everyone, so I will add this in. Great idea!